open-machine-learning-jupyter-book/deep-learning/CNN.md

```diff
@@ -13,4 +13,1201 @@ kernelspec:
   name: python3
 ---
 
-# CNN (TBD)
\ No newline at end of file
+
+# Convolutional Neural Networks
+
+
+
+```{epigraph}
+Thanks to Convolutional Neural Network, computer vision is working far better than just two years ago, and this is enabling numerous exciting applications ranging from safe autonomous driving, to accurate face recognition, to automatic reading of radiology images.
+
+-- Andrew Ng
+```
+
+
+
+Convolutional Neural Networks (CNNs) are responsible for the latest major breakthroughs in image recognition in the past few years.
+
+In mathematics, a convolution is a function which is applied over the output of another function. In our case, we will consider applying a matrix mutliplication (filter) across an image. See the below diagram for an example of how this may work.
+
+
+
+:::{figure-md} 01_intro_cnn-dl
+<img src="../../images/deep-learning/CNN/01_intro_cnn.png" width="90%" class="bg-white mb-1">
+
+Illustration of matrix mutliplication (filter) in CNN {cite}`reluwiki`
+:::
+
+
+
+<iframe src="../html-iframes/conv-demo/index.html" width="105%" height="700px;" style="border:none;"></iframe>

```

Comment:
Do you want to give any description to this iframe section? As I can't see the screenshot, I don't know what this is for. E.g. you can include it to a ```{seealso} section if it is a further reading staff.